src/internal/operations/backup_test.go

```diff
@@ -504,7 +504,7 @@ func (suite *BackupOpIntegrationSuite) TestBackup_Run_sharePoint() {
 		sel    = selectors.NewSharePointBackup()
 	)
 
-	sel.Include(sel.Sites([]string{siteID}))
+	sel.Include(sel.Libraries([]string{siteID}, selectors.Any()))
```

Comment:
Can this get a TODO about it needing to switch back to `Sites` after implementation?

Reply:
TODO added in [3933622](https://github.com/alcionai/corso/pull/1786/commits/3933622db94e3b6967b82c4fdba10e42c2f59628)